source/administration/upgrade.rst

```diff
@@ -1,120 +1,75 @@
+  "3.0.3", "See `Important Notices`_ 1 to 7","`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
+  "2.2.0", "Not Applicable", "`Upgrading to version 3.0 <../administration/upgrading-to-3.0.html>`_"
+  "2.1.0", "Not Applicable", "`Upgrading to version 3.0 <../administration/upgrading-to-3.0.html>`_"
+  "2.0.0", "Not Applicable", "`Upgrading to version 3.0 <../administration/upgrading-to-3.0.html>`_"
+  "1.4.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.3.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.2.1", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.2.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.1.1", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.1.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+  "1.0.0", "Not Applicable", "`Upgrading to version 2.0 <../administration/upgrading-to-2.0.html>`_"
+
+.. toctree::
+   :maxdepth: 1
+
+   /administration/upgrading-to-latest.rst
+   /administration/upgrading-to-3.0.rst
+   /administration/upgrading-to-2.0.rst
 
 Important Notices
 -----------------
 
-1. Security related changes were made in 3.9.0 that cause any previously created team invite links, password reset links, and email verification links to no longer work. You must update any place where you have published these links.
+1. After upgrading to version 3.4: If public links are enabled, existing public links will no longer be valid. This is because in earlier versions, existing public links were not invalidated when the Public Link Salt was regenerated. You must update any place where you have published these links.
 
-2. Security-related changes were made in 3.7.5 and 3.8.0 that require you to verify settings in the System Console before upgrading from version 3.7.4 and earlier.
+2. Enterprise Edition only: After upgrading to version 3.6.0, if you previously had values set for *RestrictPublicChannelManagement* and *RestrictPrivateChannelManagement*, the new settings for *RestrictPublicChannelCreation*, *RestrictPrivateChannelCreation*, *RestrictPublicChannelDeletion*, and *RestrictPrivateChannelDeletion* will take those settings as their default values.
 
-  1. In the GENERAL section of the System Console, click **Configuration** and make sure that the **Site URL** is specified. It must not be empty. For more information about SiteURL, see `Configuration Settings <config-settings.html#site-url>`_
-  2. In the GENERAL section of the System Console, click **Logging** and make sure that the **File Log Directory** field is either empty or has a directory path only. It must not have a filename as part of the path.
+3. In version 3.6, update the maximum number of files that can be open:
```

Comment:
"update the maximum number of files that can be open" seems confusing. Do we need to describe it in this way?

Also, does this step require action or is it just verifying? The two verbs are conflicting "update" and "verify"